project.json

```diff
@@ -1,16 +1,17 @@
 {
     "project_name": "MultiplayerSample",
-    "product_name": "MultiplayerSample",
-    "executable_name": "MultiplayerSampleLauncher",
-    "modules": [],
     "project_id": "{16D27830-D7FF-4124-B20B-2CF0387E8C7D}",
-    "android_settings": {
-        "package_name": "com.o3de.yourgame",
-        "version_number": 1,
-        "version_name": "1.0.0.0",
-        "orientation": "landscape"
-    },
-    "xenia_settings": {},
-    "provo_settings": {},
-    "engine": "o3de"
-}
\ No newline at end of file
+    "origin": "The primary repo for MultiplayerSample goes here: i.e. http://www.mydomain.com",
+    "license": "What license MultiplayerSample uses goes here: i.e. https://opensource.org/licenses/Apache-2.0 Or https://opensource.org/licenses/MIT etc.",
+    "display_name": "MultiplayerSample",
+    "summary": "A short description of MultiplayerSample.",
```

Comment:
Should these be updated? The summary could just be pulled from the README, "A simple third-person multiplayer sample for O3DE." This could also be a separate follow up PR.

Reply:
I can update it right now.
It is doesn't take any time at all